torch/fx/experimental/proxy_tensor.py

```diff
@@ -303,6 +304,29 @@ def __torch_dispatch__(cls, func_overload, types, args=(), kwargs=None):
                     break
         assert proxy_mode is not None, "At least one argument must be a ProxyTensor"
 
+        if func_overload.__name__ == 'torch.cond':
+            # TODO: need to parse
+            assert kwargs is None or not kwargs
+            pred, true_fn, false_fn, *operands = args
+
+            true_graph = get_isolated_graphmodule(true_fn, *operands, {})
```

Comment:
Didn't know this existed -- this is cool

Reply:
It was added a week ago 🙂

Reply:
Yea, thanks @IvanYashchuk it works well!